monitor.rb

```diff
@@ -72,7 +72,7 @@ def process_new_users
       @cf_client.add_user_to_org(user["metadata"]["guid"], sandbox_org['metadata']["guid"])
       # create user space using the first portion of the email address as the space name
       @cf_client.create_space(user_space_name, sandbox_org['metadata']["guid"],
-          [user["metadata"]["guid"]], [user["metadata"]["guid"]], sandbox_org['metadata']["space_quota_guid"])
+          [user["metadata"]["guid"]], [user["metadata"]["guid"]], sandbox_org['entity']['quota_definition_guid'])
```

Comment:
Are you sure we need the quota for the org here? I think we need the quota for the space....
Maybe I don't get this